content/work/cfoshare.md

```diff
@@ -27,11 +27,23 @@ summary: |
   migrating content,
   optimizing responsive images,
   and training CFOshare staff in continued maintenance.
+  CFOshare is a "fractional CFO"
+  that serves as an outsourced finance 
+  and accounting department
+  for a range of businesses.
 ---
 
-CFOshare is a "fractional CFO"
-that serves as an outsourced finance and accounting department
-for a range of businesses.
+**Dates**: 2018 - 2020 
+
+**Sector**: Finance & Accounting
+
+**What We Did**: 
+- Transitioned website to WordPress CMS
+- Brand implementation
+- Content migration
+- Image optimization
+- Trained staff in on-going maintenance  
+
```

Comment:
What if we did something here like we did on the Design Services page (I plopped this content in this space, but we'd use another image)![Screen Shot 2021-07-08 at 3 52 09 PM](https://user-images.githubusercontent.com/1581694/124989789-224f1b00-e005-11eb-8c97-44b951ba9f14.jpg)


Orignial can be found here:
https://deploy-preview-167--oddleventy.netlify.app/services/development/

The original section looks like this in code if this is helpful:

```
{% import 'embed.macros.njk' as embed %}
{% import 'layout.macros.njk' as layout %}


{{ layout.title('Services') }}

{% call embed.media_block(
  media=embed.screen(
    notch='0',
    content=embed.img(
      src='pages/trig-phone.jpg',
      alt='trig race screen',
      width=744 ,
      height=1611,
      sizes='media'
    )
  )
) %}

{% call layout.grid('narrow-columns') %}

{% call layout.block('column') %}

### Design

- User Experience (UX) - OOUX Certified
- User Interface (UI) & Interaction
- Logos & Branding
- Integrated Design Systems
- Usability Testing
- Information Architecture

{% endcall %}

{% call layout.block('column') %}

### Development

- Full-stack Frontend & Backend
- 100% Test Coverage
- Full Documentation
- Automated Pattern Libraries
- WCAG Accessibility
- Refactors & Performance

{% endcall %}

{% endcall %}

{% endcall %}
```